bundles/org.openhab.binding.intesis/src/main/java/org/openhab/binding/intesis/internal/handler/IntesisHomeHandler.java

```diff
@@ -86,6 +87,8 @@ public class IntesisHomeHandler extends BaseThingHandler {
 
     private IntesisHomeConfiguration config = new IntesisHomeConfiguration();
 
+    private String m_sessionId = "";
```

Comment:
```suggestion
    private String sessionId  = "";
```

SAT Error: Name 'm_sessionId' must match pattern '^[a-z][a-zA-Z0-9]*$'.

Reply:
Hi @lsiepel,
Thank you for the fasst review!
What means "SAT Error"?
I wanted to make clear what is the global variable and what is a local one. But if you think this is not reasonable, it is OK for me. So I check in an updated version.

Reply:
Details can be found here: https://www.openhab.org/docs/developer/guidelines.html#static-code-analysis
The report is available in your local build, but also on this PR you can click details next to the ci build check, click summary and the download the sat-summary-report : https://github.com/openhab/openhab-addons/actions/runs/8154201322?pr=16476